lib/pbench/agent/results.py

```diff
@@ -313,7 +320,7 @@ def __init__(
 
     def copy_result_tb(
         self, token: str, access: Optional[str] = None, metadata: Optional[List] = None
-    ) -> None:
+    ) -> requests.models.Response:
```

Comment:
While I think that's technically correct, it's more common (and a bit more compact) to use `requests.Response`.

Reply:
OK